app/css/openmrs-addonmanager.css

```diff
@@ -192,3 +192,51 @@ button h5{
     background-color:#dddddd;
     border: 0px;
 }
+
+#manageApps{
```

Comment:
What is the convention here? "manageApps" or "manage-apps"?

Reply:
Okay Daniel, it's an Id, I thought of having it as manageApps since it's not a class. But I will rectify this now.

Reply:
If an id then fine.